base_site/index.html

```diff
@@ -57,78 +69,91 @@ <h1 class="heading-style-h1 text-color-gray">Technical Documentation</h1>
                     <div class="spacer-small"></div>
-                    <a href="https://docs.quantinuum.com/nexus/index.html" class="product_link-wrap w-inline-block">
+                    <a href="https://docs.quantinuum.com/nexus/examples/overview.html" class="product_link-wrap w-inline-block">
                       <p class="product_link-text">Tutorials and Documentation</p>
-                      <p class="product_description is-list">Find the latest technical documentation and  additional resources</p>
+                      <p class="product_description is-list">Read the full Quantinuum Nexus documentation</p>
                     </a>
                   </div>
-                  <div id="w-node-c6173a0f-f663-21c6-62d6-04d227112a72-c2a09c1d" class="product-list_image-wrapper is-nexus"><img src="images/nexus-sized.png" loading="lazy" alt="" class="product-list_image"></div>
                 </div>
                 <div class="w-layout-grid product-list_component">
                   <div class="product-list_content">
-                    <div class="product_logo-wrap"><img src="images/h-series.svg" loading="lazy" alt="" class="image-fit-contain"></div>
+                    <a href="https://docs.quantinuum.com/h-series/" class="product_logo-wrap w-inline-block"><img src="_static/images/h-series.svg" loading="lazy" alt="" class="image-fit-contain"></a>
                     <div class="spacer-small"></div>
-                    <p class="product_description">The world’s highest-performing  quantum hardware</p>
+                    <p class="product_description">The world’s highest-performing quantum hardware</p>
                     <div class="spacer-medium"></div>
-                    <a href="https://docs.quantinuum.com/h-series/user_guide/hardware_user_guide/access.html" class="product_link-wrap w-inline-block">
-                      <p class="product_link-text">Get Started with H-Series</p>
-                      <p class="product_description is-list">Explore how to use the industry&#x27;s leading  quantum processors</p>
+                    <a href="https://docs.quantinuum.com/h-series/user_guide/hardware_user_guide/access" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">H-Series User Guide</p>
+                      <p class="product_description is-list">Explore how to use the industry&#x27;s leading quantum processors</p>
                     </a>
                     <div class="spacer-small"></div>
-                    <a href="https://docs.quantinuum.com/h-series/index.html" class="product_link-wrap w-inline-block">
-                      <p class="product_link-text">Tutorials and Documentation</p>
-                      <p class="product_description is-list">Find the latest technical documentation and  additional resources</p>
+                    <a href="https://docs.quantinuum.com/h-series/trainings/getting_started/index" class="product_link-wrap w-inline-block">
```

Comment:
```suggestion
                    <a href="https://docs.quantinuum.com/h-series/trainings/getting_started/index.html" class="product_link-wrap w-inline-block">
```

(it works either way but this is clearer about exactly where the link's going)